cdap-ranger/cdap-ranger-binding/src/main/resources/ranger-cdap-audit.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+  <property>
+    <name>xasecure.audit.is.enabled</name>
+    <value>true</value>
+  </property>
+
+  <!-- DB audit provider configuration -->
+  <property>
+    <name>xasecure.audit.db.is.enabled</name>
+    <value>false</value>
+  </property>
+
+  <property>
+    <name>xasecure.audit.db.is.async</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>xasecure.audit.db.async.max.queue.size</name>
+    <value>10240</value>
+  </property>
+
+  <property>
+    <name>xasecure.audit.db.async.max.flush.interval.ms</name>
+    <value>30000</value>
+  </property>
+
+  <property>
+    <name>xasecure.audit.db.batch.size</name>
+    <value>100</value>
+  </property>
+
+  <!--  Properties whose name begin with "xasecure.audit.jpa." are used to configure JPA -->
+  <property>
+    <name>xasecure.audit.jpa.javax.persistence.jdbc.url</name>
+    <value>jdbc:mysql://localhost:3306/ranger_audit</value>
```

Comment:
I don't understand this comment. This is ranger side configuration. We don't need to handle anything here. User is responsible for configuring ranger correctly. If the configured user does not have privilege for ranger_audit then ranger will fail to write audit logs